synapse/tests/test_lib_spawn.py

```diff
@@ -514,11 +515,13 @@ async def test_stormpkg(self):
         otherpkg = {
             'name': 'foosball',
             'version': (0, 0, 1),
+            'minversion': (2, 8, 0),
         }
 
         stormpkg = {
             'name': 'stormpkg',
-            'version': (1, 2, 3)
+            'version': (1, 2, 3),
+            'minversion': (2, 8, 0),
```

Comment:
it's a bit cumbersome, but it may be way more self-explanatory if we named this `synapse_minversion` or something. Without a bit more context i could see it being confusing.